README.md

```diff
@@ -53,6 +53,10 @@ selecting the `Use DiCy` option in the settings page. [DiCy][] will be installed
 automatically and so no further action is required for either TeX Live or
 MiKTeX.
 
+### Code Highlighting
+In order to properly work in Atom with this package, you'll need a code
+highlighting extension, namely [`language-latex`][language-latex-pkg].
+
```

Comment:
Additionally, upon discussing this with @yitzchak, we think it makes sense to recommend using the language-knitr package as well. Maybe an addition to the above, along the lines of:
``` md
Additionally, you might want to consider installing
[`language-knitr`][language-knitr] if you want to take advantage of the
provided knitr support.
``` 

@yitzchak Should we also mention language-r perhaps? Not strictly required, but highly recommended in combination with language-knitr.